Okta/OktaAuth/OktaAuthStateManager.swift

```diff
@@ -157,33 +157,51 @@ open class OktaAuthStateManager: NSObject, NSCoding {
 }
 
 public extension OktaAuthStateManager {
-    
-    private static let secureStorageKey = "OktaAuthStateManager"
 
-    class func readFromSecureStorage() -> OktaAuthStateManager? {
-        guard let encodedAuthState: Data = try? OktaKeychain.get(key: secureStorageKey) else {
-            return nil
-        }
-
-        guard let state = NSKeyedUnarchiver.unarchiveObject(with: encodedAuthState) as? OktaAuthStateManager else {
-            return nil
+    private static let legacySecureStorageKey = "OktaAuthStateManager"
```

Comment:
nit: can we move this constant to `readFromSecureStorage()` function? Just to define where it is required

Reply:
Fixed.